src/utils/access-token-store.util.ts

```diff
@@ -0,0 +1,15 @@
+const ACCESS_TOKEN_KEY = 'accessToken';
+
+export class AccessTokenStore {
+  static getToken() {
+    return localStorage.getItem(ACCESS_TOKEN_KEY);
+  }
+
+  static setToken(token: string) {
+    localStorage.setItem(ACCESS_TOKEN_KEY, token);
```

Comment:
Set the expiration time, which was provided by the backend.

Reply:
The backend already does a bunch of checks though. Is it necessary? if so, should I create a separate `tokenExpiration` item and store the date there?

Reply:
Yes. The browser automatically deletes the token when it is expired if the expiration time is set. So you won't have to send a request just to find out that token is expired and needs to be refreshed. Instead, you can refresh the token if it is not in cookies anymore.
It just helps to improve performance, not really required. You can leave it if you want.

Reply:
> The backend already does a bunch of checks though. Is it necessary? if so, should I create a separate `tokenExpiration` item and store the date there?

Yes. The browser automatically deletes the token when it is expired if the expiration time is set. So you won't have to send a request just to find out that token is expired and needs to be refreshed. Instead, you can refresh the token if it is not in cookies anymore.
It just helps to improve performance, not really required. You can leave it if you want.

Reply:
fixed